src/go/rpk/pkg/cli/cmd/acl/list.go

```diff
@@ -295,21 +291,21 @@ func printResourceACLs(resACLs []sarama.ResourceAcls) {
 	t := ui.NewRpkTable(log.StandardLogger().Out)
 	t.SetColWidth(80)
 	t.SetAutoWrapText(true)
-	t.SetHeader([]string{"Principal", "Host", "Operation", "Permission Type", "Resource Type", "Resource Name"})
+	t.SetHeader([]string{"Principal", "Host", "Operation", "Permission Type", "Resource Type", "Resource Name", "Name Pattern"})
```

Comment:
This can just be `Name`, not `Name Pattern`.

Reply:
"Name" could be confused with the "Resource Name"

Reply:
Wait, looking at the usage below: why not `Resource Pattern Type`?

Reply:
That works. I was looking for shorter alternatives though.

Reply:
fwiw kcl takes the following approach: `TYPE\tNAME\tPATTERN\tPRINCIPAL\tHOST\tOPERATION\tPERMISSION\tERROR\tERROR MESSAGE\n`

I think `Resource Pattern` or `Pattern` may also be explicit enough? Not sure how necessary `Type` is in most of those fields.